### ISSUE TITLE
download progress + cancel possibility when waiting

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#421]
Converted from [SourceForge issue 2889442](http://sourceforge.net/support/tracker.php?aid=2889442), submitted by albertzeyer

When the game (server) started and you (client) are waiting for a file download, there should be some progress bar or at least some hint that it is downloading right now. And there should also be a possibility to cancel. Right now, if it would take very long there, perhaps because of timeout or whatever, it would hang there forever.

**Comment by albertz** [comment]
Submitted by dark_charlie

There already is a progressbar for that. See CClient_Draw.cpp line 573. If it doesn't show it, it's a different bug somewhere.

**Comment by albertz** [comment]
Submitted by dark_charlie

And you can cancel it by leaving the server.

**Comment by albertz** [comment]
Submitted by albertzeyer

Hm, so it stays in lobby when server started and you are not ready with downloading?

I got the "feeling" that it wasn't in any menu at all. "Feeling" = because my download was just too fast, took about 100ms to download, so I couldn't really tell. I only saw the map loading animation all the time. First I thought that it retried every single frame to load the map, because I had the feeling the map loading animation was longer than usual.
